kroxylicious-systemtests/src/main/resources/nonJVMClient_overrides.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Can this be applied to any version of Openshift or do we need to gate it on being 4.18+ ?

Reply:
Testing it